xml/ns-System.IO.Pipelines.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.IO.Pipelines">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>The System.IO.Pipelines namespace provides types that allow performing complex, high performance IO operations.</summary>
```

Comment:
```suggestion
    <summary>The System.IO.Pipelines namespace provides types for performing complex, high performance input-output (IO) operations.</summary>
```

Reply:
Oh GitHub didn't refresh the page. I only saw your signoff but didn't see your suggestion. I'll submit a new PR addressing it, @gewarren.